src/ucp/rma/flush.c

```diff
@@ -267,7 +268,8 @@ ucs_status_ptr_t ucp_ep_flush_internal(ucp_ep_h ep, unsigned uct_flags,
     req->send.ep                = ep;
     req->send.cb                = req_cb;
     req->send.flush.flushed_cb  = flushed_cb;
-    req->send.flush.lanes       = UCS_MASK(ucp_ep_num_lanes(ep));
+    req->send.flush.lanes       = UCS_MASK(ucp_ep_num_lanes(ep)) &
+                                  ~UCS_BIT(cm_lane);
```

Comment:
check if cm_lane is valid

Reply:
this works as well if it's not valid (NULL_LANE == 255)

Reply:
seems hack, better check explicitly